internal/pkg/cli/create_test.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 func TestInteractiveCreateConfig(t *testing.T) {
+	t.Parallel()
```

Comment:
V `Go` sa musi oznacit, ze test moze beza paralelne, u nas su to teraz uz vsetky testy (okrem jedneho).